plugins/tutor-contrib-paragon/tutorparagon/patches/openedx-lms-development-settings

```diff
@@ -0,0 +1,23 @@
+{% if not PARAGON_ENABLED_THEMES %}{% set PARAGON_ENABLED_THEMES = ['light'] %}{% endif %}
+{% set PROTOCOL = 'https' if ENABLE_HTTPS else 'http' %}
+{% set BASE_URL = PROTOCOL ~ "://" ~ "localhost" ~ ":" ~ PARAGON_STATIC_SERVER_PORT ~ "/" %}
```

Comment:
This seems to be the only line that differs between this file and `openedx-lms-production-settings`. Would it be possible to make a shared template and pull that in to each of these?